pkgs/by-name/qm/qmmp/package.nix

```diff
@@ -97,7 +95,9 @@ stdenv.mkDerivation rec {
     game-music-emu
     libarchive
     opusfile
-    soxr
+    (soxr.overrideAttrs (oldAttrs: {
+      outputs = [ "out" ];
+    }))
```

Comment:
You can rebase on master now and remove this hunk. (soxr has now fixed the include directory in its pkg-config file.)